### PR TITLE
Lendingarsidur texts

### PR DESCRIPTION
**Pull request by 17luke**
@thdg Can you review? These are the changes requested by Anna and Bjarni.

### CODE REVIEW

File: public/static/locales/isl/l2.json [hunk 1]
```diff
@@ -1,5 +1,8 @@
...
+    "l2-explanation-long": "Þessi söfnun hefur sem markmið að auka fjölbreytni í íslenskum máltæknigögnum. Máltækni hefur náð langt á undanförnum árum en mörg talgreiningarkerfi eiga enn erfitt með að skilja íslensku innflytjenda. Við hvetjum alla sem eiga íslensku sem annað tungumál að taka þátt, jafnt byrjendum sem þeim sem tala reiprennandi íslensku. Þetta safn ætti að ná yfir alls kyns íslensku.",
+    "l2-september-prize-intro": "Gjafir fyrir sjálfboðaliða í september:",
+    "l2-september-prize-300": "Fólk sem tekur upp yfir 300 setningar í september fær 1 mánuð ókeypis áskrift hjá Storytel og 1 almennt gjafabréf í Smárabíó.",
+    "l2-september-prize-700": "Fólk sem tekur upp yfir 700 setningar í septemberfær 1 mánuð ókeypis áskrift hjá Storytel og 5.000 kr. Dineout gjafakort."
```
Comment (SmariF89) [inline public/static/locales/isl/l2.json:7]: - Fólk sem tekur upp **meira en**...
- "septemberfær" -> "september fær"
- "1 mánuð ókeypis áskrift" -> "eins mánaðar ókeypis áskrift"
- "1 almennt" -> "eitt almennt"

File: public/static/locales/isl/parallel.json [hunk 2]
```diff
@@ -0,0 +1,7 @@
+{
+    "sign-in-hello": "Þú getur fylgst með framvindu þinni með því að búa til aðgang.",
+    "sign-in-hello-link": "Smelltu hér til að skrá þig inn eða búa til aðgang.",
+    "google-form-link": "Smelltu hér til að fylla út GoogleForms-skjalið.",
```
Comment (SmariF89) [inline public/static/locales/isl/parallel.json:4]: "GoogleForms" -> "Google Forms"

File: src/components/subpages/parallel-sign-in-hello.tsx [hunk 3]
```diff
@@ -23,17 +23,53 @@ export const ParallelSignInHello: React.FC<Props> = (props) => {
...
             <SignInContainer>
-                Vinsamlegast skráðu þig inn til að taka þátt.
+                Samhliða safn til framburðarannsókna er máltækniverkefni sem
+                snýst um að safna hljóðupptökum frá fólki sem talar íslensku sem
+                annað mál og einnig frá fólki sem hefur íslensku sem móðurmál.
+                Þessar hljóðupptökur verða notaðar til að búa til gagnasafn
+                fyrir rannsóknir og þróun á máltækniforritum. Þátttakendur lesa
+                setningar undir Samhliða safn til framburðarannsókna á Samrómi
+                og svara nokkrum spurningum um aldur, kyn og tungumálakunnáttu í
+                GoogleForms-skjali.
```
Comment (SmariF89) [inline src/components/subpages/parallel-sign-in-hello.tsx]: Use `t`?

File: src/components/subpages/parallel-sign-in-hello.tsx [hunk 4]
```diff
@@ -23,17 +23,53 @@ export const ParallelSignInHello: React.FC<Props> = (props) => {
...
+                        </Link>
+                    </li>
+                    <li>
+                        <Link
+                            href={'https://forms.gle/bwHyc332vbmJa1VT7'}
+                            passHref
+                        >
+                            <StyledLink>
+                                Smelltu hér til að fylla út GoogleForms-skjalið.
+                            </StyledLink>
```
Comment (SmariF89) [inline src/components/subpages/parallel-sign-in-hello.tsx]: Ditto.

File: src/components/subpages/parallel-sign-in-hello.tsx [hunk 5]
```diff
@@ -23,17 +23,53 @@ export const ParallelSignInHello: React.FC<Props> = (props) => {
...
+    const [clientRendered, setClientRendered] = React.useState(false);
+
+    React.useEffect(() => {
+        // Set client-rendered state after component mounts
+        setClientRendered(true);
+    }, []);
+
+    if (!clientRendered) {
+        return null;
+    }
```
Comment (SmariF89) [inline src/components/subpages/parallel-sign-in-hello.tsx:38]: Why is this necessary?

Comment (17luke) [reply src/components/subpages/parallel-sign-in-hello.tsx:38]: There's a bug where the format is very strange if you access the page directly URL. This workaround was already done for the L2 subpage but not yet for the parallel subpage. So in the current deployment, you can see the bug if you go directly to https://samromur.is/parallel

File: public/static/locales/isl/l2.json [hunk 6]
```diff
@@ -1,4 +1,5 @@
 {
     "sign-in-hello": "Þú getur fylgst með framvindu þinni með því að búa til aðgang.",
-    "sign-in-hello-link": "Smelltu hér til að skrá þig inn eða búa til aðgang."
+    "sign-in-hello-link": "Smelltu hér til að skrá þig inn eða búa til aðgang.",
+    "l2-explanation-long": "Þessi söfnun hefur sem markmið að auka fjölbreytni í íslenskum máltæknigögnum. Máltækni hefur náð langt á undanförnum árum en mörg talgreiningarkerfi eiga enn erfitt með að skilja íslensku innflytjenda. Við hvetjum alla sem eiga íslensku sem annað tungumál að taka þátt, jafnt byrjendum sem þeim sem tala reiprennandi íslensku. Þetta safn ætti að ná yfir alls kyns íslensku."
```
[the remark this answers is not in the document]
Comment (17luke) [reply public/static/locales/isl/l2.json]: Changed in 701afa8f2c3971aaca11d7a957262b124fd5f9b2

File: public/static/locales/isl/parallel.json (hunk 2 above)
Comment (17luke) [reply public/static/locales/isl/parallel.json]: Changed in 701afa8f2c3971aaca11d7a957262b124fd5f9b2

File: public/static/locales/isl/parallel.json [hunk 7]
```diff
@@ -0,0 +1,7 @@
...
+    "to-take-part": "Til að taka þátt:",
+    "parallel-explanation-long": "Samhliða safn til framburðarannsókna er máltækniverkefni sem snýst um að safna hljóðupptökum frá fólki sem talar íslensku sem annað mál og einnig frá fólki sem hefur íslensku sem móðurmál. Þessar hljóðupptökur verða notaðar til að búa til gagnasafn fyrir rannsóknir og þróun á máltækniforritum. Þátttakendur lesa setningar undir Samhliða safn til framburðarannsókna á Samrómi og svara nokkrum spurningum um aldur, kyn og tungumálakunnáttu í GoogleForms-skjali."
```
[the remark this answers is not in the document]
Comment (17luke) [reply public/static/locales/isl/parallel.json]: Changed in 701afa8f2c3971aaca11d7a957262b124fd5f9b2

File: src/components/subpages/parallel-sign-in-hello.tsx (hunk 3 above)
Comment (17luke) [reply src/components/subpages/parallel-sign-in-hello.tsx]: Think this comment refers to a placeholder in an older commit. Is t in the final few commits.

File: src/components/subpages/parallel-sign-in-hello.tsx (hunk 4 above)
Comment (17luke) [reply src/components/subpages/parallel-sign-in-hello.tsx]: Changed in 701afa8f2c3971aaca11d7a957262b124fd5f9b2

**Comment by 17luke** [comment]
Thanks! I've made all those changes except the prettier formatting. Prettier format check is only failing on some other files in the repo that seem to be a few months old. I'll fix them in a separate PR.

Excerpt from Prettier format check log:
> [warn] src/components/contribute/setup/contribute.tsx
> [warn] src/components/contribute/setup/demographic-form.tsx
> [warn] src/pages/api/contribute/upload.ts
> [warn] src/pages/api/stats/user-l2.ts
> [warn] src/server/database/sentences.ts
> [warn] src/services/contribute-api.ts
> [warn] src/services/stats-api.ts
> [warn] src/store/user/reducer.ts
> [warn] src/utilities/subpage-helper.ts

**Comment by 17luke** [comment]
Separate PR for prettier format check:
https://github.com/cadia-lvl/samromur/pull/155

File: public/static/locales/isl/l2.json [hunk 8]
```diff
@@ -1,8 +1,8 @@
 {
     "sign-in-hello": "Þú getur fylgst með framvindu þinni með því að búa til aðgang.",
     "sign-in-hello-link": "Smelltu hér til að skrá þig inn eða búa til aðgang.",
-    "l2-explanation-long": "Þessi söfnun hefur sem markmið að auka fjölbreytni í íslenskum máltæknigögnum. Máltækni hefur náð langt á undanförnum árum en mörg talgreiningarkerfi eiga enn erfitt með að skilja íslensku innflytjenda. Við hvetjum alla sem eiga íslensku sem annað tungumál að taka þátt, jafnt byrjendum sem þeim sem tala reiprennandi íslensku. Þetta safn ætti að ná yfir alls kyns íslensku.",
+    "l2-explanation-long": "Þessi söfnun hefur sem markmið að auka fjölbreytni í íslenskum máltæknigögnum. Máltækni hefur náð langt á undanförnum árum en mörg talgreiningarkerfi eiga enn erfitt með að skilja íslensku innflytjenda. Við hvetjum alla sem hafa íslensku sem annað tungumál að taka þátt, jafnt byrjendum sem og þeir sem tala reiprennandi íslensku. Þetta safn ætti að ná yfir alls kyns íslensku.",
```
Comment (SmariF89) [inline public/static/locales/isl/l2.json:4]: byrjendum -> byrjendu**r**

File: src/components/subpages/l2-sign-in-hello.tsx [hunk 9]
```diff
@@ -43,6 +43,15 @@ export const L2SignInHello: React.FC<Props> = (props) => {
         <L2Container>
             {!isAuthenticated && (
                 <HelloContainer>
+                    Þessi söfnun hefur sem markmið að auka fjölbreytni í
+                    íslenskum máltæknigögnum. Máltækni hefur náð langt á
+                    undanförnum árum en mörg talgreiningarkerfi eiga enn erfitt
+                    með að skilja íslensku innflytjenda. Við hvetjum alla sem
+                    eiga íslensku sem annað tungumál að taka þátt, jafnt
```
[the remark this answers is not in the document]
Comment (SmariF89) [reply src/components/subpages/l2-sign-in-hello.tsx]: Oh, you're right. Sorry. I was doing a per-commit review and oversaw this.

File: src/components/subpages/parallel-sign-in-hello.tsx (hunk 3 above)
Comment (SmariF89) [reply src/components/subpages/parallel-sign-in-hello.tsx]: Right.

File: public/static/locales/isl/l2.json (hunk 8 above)
Comment (17luke) [reply public/static/locales/isl/l2.json]: Æi yep good catch! Fixed in [9b6d763](https://github.com/cadia-lvl/samromur/pull/153/commits/9b6d763dcc19d3e353a23861ce97322aba2cc9aa)

**Comment by 17luke** [comment]
Think all is resolved now
